src/material-examples/autocomplete-overview/autocomplete-overview-example.ts

```diff
@@ -73,7 +73,7 @@ export class AutocompleteOverviewExample {
   }
 
   filterStates(val: string) {
-    return val ? this.states.filter(s => new RegExp(`^${val}`, 'gi').test(s))
+    return val ? this.states.filter(s => s.toLowerCase().includes(val.toLowerCase()))
```

Comment:
`includes` is not supported in IE11 (which we still support). We try to avoid requiring polyfills where possible